src/constants.ts

```diff
@@ -15,6 +15,12 @@ export const ONE_SECOND_MS = 1000;
 export const ONE_MINUTE_MS = ONE_SECOND_MS * 60;
 export const TEN_MINUTES_MS = ONE_MINUTE_MS * 10;
 
+// Mesh can't handle a large amount of orders over websocket. If the orders submitted are greater than
+// MESH_WEBSOCKET_ORDER_BATCH_LIMIT, we send via HTTP instead
+export const MESH_WEBSOCKET_ORDER_BATCH_LIMIT = 50;
+export const MESH_WEBSOCKET_DEFAULT_ORDER_BATCH_SIZE = 10;
+export const MESH_HTTP_DEFAULT_ORDER_BATCH_SIZE = 100;
```

Comment:
@xianny how were these numbers chosen? They all seem quite low.

Reply:
Also, why not just send all of them over HTTP all the time? It seems like it'll always be more efficient given these settings.

Reply:
Yeah, sending them over HTTP all the time would make sense. The batch size is kind of a stab in the dark TBH. I think ultimately we have to chunk by size of actual bytes, not number of orders. Setting it to 300 and sending all over HTTP for now.